packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -478,6 +478,12 @@ export interface DatabaseInstanceNewProps {
    * @default RemovalPolicy.Retain
    */
   readonly removalPolicy?: RemovalPolicy
+
+  /**
+   * Upper limit to which RDS can scale the storage
+   * @default - No default value
```

Comment:
Explain what this default actually means. Does it mean that the RDS instance does not scale, or scale as much as the underlying hardware allows it?